### ISSUE TITLE
Unable to add node driver 

### ISSUE THREAD

**Issue by daxmc99** [rancher/rancher#20796]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug QA (found on e0a97f4bbb40b1e3f884a86596183c29a40ba24c) 

**Steps to reproduce (least amount of steps as possible):**
Try to add a custom driver via the UI
Ex: the linode driver 
`https://github.com/linode/docker-machine-driver-linode/releases/download/v0.1.8/docker-machine-driver-linode_linux-amd64.zip`
UI: `https://linode.github.io/rancher-ui-driver-linode/releases/v0.2.0/component.js`

**Result:**

```
2019/06/11 20:26:41 [INFO] Copying management-state/machine-drivers/18206b37b5529c1138ddc9fe568a803b03ae3c39ebb8abe4ae1934aef49ba4bc-docker-machine-driver-linode => /opt/drivers/management-state/bin/docker-machine-driver-linode-tmp
2019/06/11 20:26:41 [ERROR] NodeDriverController nd-md7mk [node-driver-controller] failed with : Driver "" not found. Do you have the plugin binary accessible in your PATH?
```
**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): e0a97f4bbb40b1e3f884a86596183c29a40ba24c 
- Installation option (single install/HA):
Docker container and local

**Comment by alena1108** [comment]
Will be available in v2.2.5-rc2

**Comment by sowmyav27** [comment]
Verified on v2.2.5-rc2

Able to add custom driver via UI - Linode
the linode driver
`https://github.com/linode/docker-machine-driver-linode/releases/download/v0.1.8/docker-machine-driver-linode_linux-amd64.zip`
UI: `https://linode.github.io/rancher-ui-driver-linode/releases/v0.2.0/component.js`

Logs:
```
2019/06/13 21:07:19 [INFO] Download https://github.com/linode/docker-machine-driver-linode/releases/download/v0.1.8/docker-machine-driver-linode_linux-amd64.zip
2019/06/13 21:07:20 [INFO] Found driver docker-machine-driver-linode
2019/06/13 21:07:20 [INFO] Copying management-state/machine-drivers/18206b37b5529c1138ddc9fe568a803b03ae3c39ebb8abe4ae1934aef49ba4bc-docker-machine-driver-linode => /opt/drivers/management-state/bin/docker-machine-driver-linode-tmp
```
UI:
<img width="1053" alt="Screen Shot 2019-06-13 at 2 08 31 PM" src="https://user-images.githubusercontent.com/26032343/59467521-bded0d80-8de4-11e9-877d-5521cc3a4f31.png">
